app/src/main/res/values-de/strings-de.xml

```diff
@@ -480,13 +480,310 @@
 	<string name="predictive_model_inner_workings">Vorhersagemodell (Erweiterte Anzeige)</string>
 	<string name="noise_model_inner_workings">Rauschmodell (Erweiterte Anzeige)</string>
 	<string name="glucose_number_from_filtered">Glukose aus gefilterten Werten</string>
-	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-in-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
+	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-zu-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
 	<string name="upload_logs">Hochladen von Logs</string>
 	<string name="short_off_text_for_switches">AUS</string>
 	<string name="short_on_text_for_switches">AN</string>
 	<string name="install_pebble_apps">Installieren Sie Pebble Apps</string>
-	<string name="sliding_24_hour_window_summary">Verwenden Sie den letzten 24 Stunden anstatt Zeit seit Mitternacht für die Statistik.</string>
+	<string name="sliding_24_hour_window_summary">Verwende die letzten 24 Stunden anstatt der Zeit seit Mitternacht für die Statistik.</string>
 	<string name="sliding_24_hour_window">Schiebefenster</string>
-	<string name="split">Teilt</string>
+	<string name="split">Teilen</string>
 	<string name="delete">Löschen</string>
+
+	<string name="ac1_estimation_dcct">A1c-Vorhersage im DCCT-Format (%)</string>
+	<string name="ac1_estimation_ifcc">A1c-Vorhersage im IFCC-Format (mmol/mol)</string>
+	<string name="acccept_remote_snoozes">Remote-Snoozes annehmen</string>
+	<string name="accept_treatments">Behandlungen annehmen</string>
+	<string name="accounts_inside_usa">Aktiviert = Dein Account und deine Follower-Apps sind aus den USA.</string>
+	<string name="accounts_outside_usa">Deaktiviert = Dein Account und deine Follower-Apps sind nicht aus den USA.</string>
+	<string name="accuracy_evaluation">Genauigkeitseinschätzung</string>
+	<string name="act_as_motion_master">Als Motion-Master arbeiten</string>
+	<string name="advanced_bluetooth_settings">Erweiterte Bluetootheinstellungen</string>
+	<string name="advanced_calibration">Erweiterte Kalibrierung</string>
+	<string name="advanced_g5_settings">Erweiterte G5-Einstellungen für seltene Situationen</string>
+	<string name="aggressive_service_restarts">Aggressive Dienstneustarts</string>
+	<string name="alert_calibration_request">Alarmieren, wenn eine Kalibrierung angefordert wird</string>
+	<string name="alert_minutes_reraise">Anzahl der Minuten, bevor derselbe Alarm nach dem Snoozen ertönt</string>
+	<string name="alert_noisy_values">Bei verrauschten Werten nach x Minuten alarmieren</string>
+	<string name="alert_repeat_minutes">Alarmwiederholungszeit</string>
+	<string name="alert_reraise_time">Alarmwiedererhebungszeit</string>
```

Comment:
Alarm-Abspiel-Intervall?
It is hard to find anything that is understandable, not the same as "repeat" and not a longer explaination what the re-raise interval is. Hm?